dotnet-source-8.yaml

```diff
@@ -0,0 +1,211 @@
+package:
+  name: dotnet-source-8
+  version: 8.0.10
+  epoch: 1
+  description: ".NET SDK"
+  copyright:
+    - license: MIT
+  resources:
+    cpu: 2
+    memory: 32Gi
+  dependencies:
+    runtime:
+      - icu
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - clang-15
+      - clang-15-default
+      - cmake
+      - curl
+      - glibc-locale-en
+      - icu-dev
+      - krb5-dev
+      - libunwind-dev
+      - llvm15
+      - llvm15-cmake-default
+      - llvm15-dev
+      - llvm15-tools
+      - lttng-ust-dev
+      - ncurses-dev
+      - openssl-dev
+      - python3
+      - samurai
+      - wolfi-base
+      - zlib-dev
+      - dotnet-bootstrap-8
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/dotnet/dotnet
+      tag: v${{package.version}}
+      expected-commit: 8922fe64a1903ed4e35e24568efb056b3e0fad43
+      destination: /home/build/src
+
+  - working-directory: /home/build/src
+    pipeline:
+      - runs: |
+          sed -i -E 's|( /p:BuildDebPackage=false)|\1 /p:EnablePackageValidation=false|' src/runtime/eng/SourceBuild.props
+          sed -i -E 's|( /p:BuildDebPackage=false)|\1 --cmakeargs -DCLR_CMAKE_USE_SYSTEM_LIBUNWIND=TRUE|' src/runtime/eng/SourceBuild.props
+          ln -sf /usr/share/dotnet-bootstrap/dotnet .dotnet
+          mkdir -p prereqs/packages/archive/
+          ln -sf /usr/share/dotnet-bootstrap/Private.SourceBuilt.Artifacts.Bootstrap.tar.gz prereqs/packages/archive/Private.SourceBuilt.Artifacts.Bootstrap.tar.gz
+      - runs: |
+          ./build.sh --clean-while-building \
+            -- \
+            /v:n \
+            /p:ContinueOnPrebuiltBaselineError=true \
+            /p:MinimalConsoleLogOutput=false \
+            /p:SkipPortableRuntimeBuild=true \
+            /p:BuildWithOnlineSources=false \
+            /p:Configuration=Debug
```

Comment:
to be reverted, added it here to have a way of build if anyone wants.

Size differences:

# release version of dotnet
du -sh /usr/share/dotnet-source/
481.2M  /usr/share/dotnet-source/

# debug version of dotnet
du -sh /usr/share/dotnet-source/
559.3M  /usr/share/dotnet-source/

Reply:
do not want, too big.